### PR TITLE
chore: polkadot 0.9.26

### PR DESCRIPTION
**Pull request by Roznovjak**

### CODE REVIEW

**Comment by lumir-mrkva** [comment]
Task linked: [CU-2p79uye Upgrade to polkadot-v0.9.26](https://app.clickup.com/t/2p79uye)

File: nft/src/lib.rs [hunk 1]
```diff
@@ -461,25 +456,25 @@ impl<T: Config> Inspect<T::AccountId> for Pallet<T> {
...
-    fn owned(who: &T::AccountId) -> Box<dyn Iterator<Item = (Self::ClassId, Self::InstanceId)>> {
+    fn owned(who: &T::AccountId) -> Box<dyn Iterator<Item = (Self::CollectionId, Self::ItemId)>> {
         Box::new(
             pallet_uniques::Pallet::<T>::owned(who)
                 .map(|(class_id, instance_id)| (class_id.into(), instance_id.into())),
         )
     }
 
-    fn owned_in_class(class: &Self::ClassId, who: &T::AccountId) -> Box<dyn Iterator<Item = Self::InstanceId>> {
+    fn owned_in_collection(class: &Self::CollectionId, who: &T::AccountId) -> Box<dyn Iterator<Item = Self::ItemId>> {
```
Comment (lumir-mrkva) [inline nft/src/lib.rs:474]: inconsistent renaming

cc @green-jay what should we do with this unique class -> collection mess?
how does it affect kodadot?

File: nft/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-nft"
-version = "3.3.0"
+version = "3.3.1"
```
Comment (lumir-mrkva) [inline nft/Cargo.toml:3]: as it stands now, this is a huge breaking change ... should be bump to 4.0.0

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

**Review by lumir-mrkva — APPROVED** [review]

File: nft/src/lib.rs (hunk 1 above)
Comment (Roznovjak) [reply nft/src/lib.rs]: We decided to rename everything to `Collection` and `Item`.

**Comment by Roznovjak** [comment]
Closing in favor of #83

**Pull request closed without merging**